tests/chainer_tests/functions_tests/normalization_tests/test_l2_normalization.py

```diff
@@ -21,6 +21,8 @@
         {'shape': (4, 3, 2, 5), 'axis': 2},
         {'shape': (4, 3, 2, 5), 'axis': 3},
         {'shape': (4, 3, 2), 'axis': (0, 1)},
+        {'shape': (4, 3, 2, 4, 3, 2, 2), 'axis': (1, 4, 3, 6)},
+        {'shape': (0, 2), 'axis': 1},
```

Comment:
`NormalizeL2.forward` needs `force_array` in order to support `len(axis) == 0`.  (Test `{'shape': (), 'axis': ()},`.)

Reply:
Thank you for pointing out. Fixed.
(Casting `eps` is needed to support ()-shape.)